src/main/java/com/google/sps/servlets/Caption.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.InputStream;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+public class Caption {
+  /*
+   * Usage: String caption = new Caption().getCaptionFromID("abcdefg");
+   * 
+   * Return null if not found.
+   */
+
+  public CaptionService captionService;
+
+  public Caption() {
+    captionService = new CaptionService();
+  }
```

Comment:
I'm not sure what is preferred as good practice but I have it a hunch that it might be preferred to do

```
public CaptionService captionService = new CaptionService;
```

and not have a custom constructor (If you don't have a constructor, there is a default constructor behavior).


I would use constructor setters if the non-test caller cares about setting specific members or if there are different overloaded constructors that behave differently.

Reply:
Done. I wasn't aware of the default constructor, thank you for bringing this up!